apps/builddao/widget/components/ButtonLink.jsx

```diff
@@ -0,0 +1,80 @@
+const StyledLink = styled.a`
+  all: unset;
+  display: inline-flex;
+  padding: 10px 20px;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+  border-radius: 8px;
+  font: 500 14px / normal;
+  transition: all 300ms;
+
+  ${(props) =>
+    props.type === "icon" &&
+    `
+    display: flex;
+    width: 40px;
+    height: 40px;
+    padding: 5px;
+    flex-shrink: 0;
+    border-radius: 50%;
+  `}
+
+  /* Colors based on variant prop */
+  background: ${(props) => {
+    switch (props.variant) {
+      case "primary":
+        return "#FFAF51";
+      case "outline":
+        return "transparent";
+      default:
+        return "#23242B";
+    }
+  }};
+
+  color: ${(props) => {
+    switch (props.variant) {
+      case "primary":
+        return "#000";
+      case "outline":
+        return "#fff";
+      default:
+        return "#CDD0D5";
+    }
+  }};
+
+  border: ${(props) =>
+    props.variant === "outline" ? "1px solid rgba(255, 255, 255, 0.20)" : ""};
+
+  /* Hover states */
+  &:hover {
+    background: ${(props) => {
+      switch (props.variant) {
+        case "primary":
+          return "#e49b48";
+        case "outline":
+          return "rgba(255, 255, 255, 0.20)";
+        default:
+          return "#17181c";
+      }
+    }};
+  }
+`;
+
+function ButtonLink({ id, children, variant, type, href, className, style }) {
```

Comment:
Renaming to ButtonLink is going to break current implementations -- it only still works because the Button code has been pushed on chain and can always be referenced, but we lose the parity with our code base.

Can this be renamed to Button? Then add a condition that checks for href and renders accordingly